VisualPinball.Unity/VisualPinball.Unity/VPT/Table/TableSerializedSoundContainer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using VisualPinball.Engine.VPT.Sound;
+using VisualPinball.Engine.VPT.Table;
+
+namespace VisualPinball.Unity
+{
+	/// <summary>
+	/// Implements a serializable Sound container that can replace the engine table container after import
+	/// </summary>
+	[Serializable]
+	public class TableSerializedSoundContainer : TableSerializedContainer<Sound, SoundData, TableSerializedSound>
+	{
+		public override void Add(Sound value)

```

Comment:
Not something to worry about right now, but I wonder if these funcs will end up being the same for all containers with the exception of the create call. We could probably figure out a way to template that too